ember/app/ui/cases/detail/work-items/index/route.js

```diff
@@ -0,0 +1,28 @@
+import Route from "@ember/routing/route";
+import { queryManager } from "ember-apollo-client";
+import getCaseQuery from "mysagw/gql/queries/get-case.graphql";
+
+export default class CasesDetailWorkItemsRoute extends Route {
+  @queryManager apollo;
+
+  async model() {
+    const model = this.modelFor("cases.detail");
+    if (typeof model === "object" && model) {
```

Comment:
Calling `this.modelFor("cases.detail")` seems to return inconsistent results, on the first call it will be a UUID string, but on subsequent ones it will be the case as an object. Not sure why